src/component/ArticleCard.jsx

```diff
@@ -0,0 +1,14 @@
+const ArticleCard = ({article}) => {
+    return <li className="article-card">
+        <p className="article-topic">{article.topic}</p>
+        <p className="article-created-at">{article.created_at}</p>
```

Comment:
Have a look at how you can make this more human readable. The date from the database is a UTC time stamp. There are plenty of libraries around that can convert this for you, or a nice little util function. (day.js is a nice one)